.travis.yml

```diff
@@ -25,3 +25,19 @@ jobs:
         on:
           repo: solvespace/solvespace
           tags: true
+    - stage: deploy
+      name: "Snap"
+      os: linux
+      dist: bionic
+      addons:
+        snaps:
+          - name: snapcraft
+            confinement: classic
+      script: ./.travis/build-snap.sh
+      deploy:
+        provider: snap
+        skip_cleanup: true
+        snap: pkg/snap/*.snap
+        channel: edge
```

Comment:
We could also try promoting tagged builds to `candidate` or `beta` directly

Reply:
`beta` seems fine